Dockerfile.ubuntu

```diff
@@ -0,0 +1,7 @@
+FROM python:3.10.6-slim
+WORKDIR /app
+RUN apt update
+RUN apt install -y wget git libgl1 libglib2.0-0
```

Comment:
Maybe it makes more sense to provide full ffmpeg since more and more people are making Aminate with WebUI.